scripts/zf_mkdist

```diff
@@ -98,7 +98,12 @@ build_and_copy_artifacts() {
     fi
 
     build_dir="${zf_tmpdir}"/build
-    rm -rf "${build_dir}"
+    find "${build_dir}" -maxdepth 1 -mindepth 1 -type d |
+      while read -r d; do
+        if [ "$(realpath "$d")" != "$(realpath "$artifact_dir")" ]; then
```

Comment:
Yes, I suggested braces because the `zf_mkdist` script tends to use them, and that is my preference (I saw it a while ago at https://google.github.io/styleguide/shellguide.html#s5.6-variable-expansion, and it contributed to my preference).

However, it is subjective: the script uses unbraced variables as well as braced ones, braces are unnecessary, and this is my preference. Thus, I approved the PR because I'm happy if Thomas drops this suggestion or applies the `-not` predicate, which seems neater to me if it works.

👨‍🎓